File: src/test/java/com/example/pages/DropdownsPage.java
package com.example.pages;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.support.ui.Select;

public class DropdownsPage extends SeleniumBasePage {

    private final By countryDropdown = By.id("country");

    public DropdownsPage(WebDriver driver) {
        super(driver);
    }

    public DropdownsPage selectCountry(String country) {
        Select select = new Select(waitForVisible(countryDropdown));
        select.selectByVisibleText(country);
        return this;
    }
}

File: python/pages/dropdowns_page.py
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import Select
from .base_page import BasePage


class DropdownsPage(BasePage):

    COUNTRY = (By.ID, "country")

    def select_country(self, country):
        select = Select(self.wait_visible(self.COUNTRY))
        select.select_by_visible_text(country)
        return self
